website/docs/reference/global-configs.md

```diff
@@ -213,7 +213,7 @@ config:
 
 ### Failing fast
 
-Supply the `-x` or `--fail-fast` flag to `dbt run` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, then dbt will terminate the connections for these still-running models.
+[Supply the `-x` or `--fail-fast` flag to `dbt run` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, then dbt will terminate the connections for these still-running models.](https://github.com/dbt-labs/docs.getdbt.com/issues/911)
```

Comment:
hi @genwhitt ! thanks for visiting our table at WTD Writing Day. I think the copy/pasta we were working on got mixed up. Let's get your real update in!